### PR TITLE
Adding more details regarding prereqs.  Calling out yarn is important

### PR DESCRIPTION
**Pull request by kensipe**
Planning on adding details to the tutorial prereqs.  It seemed useful to provide more details on prereqs and to provide context of version awareness.

Signed-off-by: Ken Sipe <kensipe@gmail.com>

### CODE REVIEW

**Comment by kensipe** [comment]
@nicks could use a review on this addition to sample app.  I will follow with additions to the tutorial.  The big miss IMO is the need for Node and yarn.  I assume all the tilt devs have it installed, but I don't think we can assume that.  I could see plenty of backend-only devs looking to use tilt.  🤞

**Comment by nicks** [comment]
cc @milas what do you think?

**Comment by kensipe** [comment]
I tested the tutorial and it does seem to require yarn... I'll provide details later this afternoon

**Comment by kensipe** [comment]
@milas Testing this morning and it does seem to need yarn, which makes the PR instructions valuable.  The alternative would be to remove the need for yarn.

## my setup
1. install tilt with homebrew
2. forked and cloned tlit-avatars
3. have docker running

## confirmation
```sh
git remote -v
origin	git@github.com:kensipe/tilt-avatars.git (fetch)
origin	git@github.com:kensipe/tilt-avatars.git (push)
upstream	git@github.com:tilt-dev/tilt-avatars.git (fetch)
upstream	git@github.com:tilt-dev/tilt-avatars.git (push)
```
HEAD of master is ca69472b06816f860c069573fbe373b7e9274dbf

**note:**  I use `nvm` which for the test is not set which results in:
```sh
node -v
zsh: command not found: node
```

the demo displays for a few mins...  
```sh
tilt demo

Hang tight while Tilt prepares your demo environment!
	Creating "tilt-demo-2765203152" local Kubernetes cluster...
```
then a screen alert that reads

Screen Shot 2022-02-11 at 7 52 52 AM

**Comment by milas** [comment]
Thanks for the detailed report, @kensipe!

You must have built Tilt from source! When built from source, Tilt defaults to `--web-mode=dev` where it does a `yarn install && yarn run start` (see [pkg/assets/dev.go](https://github.com/tilt-dev/tilt/blob/0037160b25972a94229a268163536cb2717081e4/pkg/assets/dev.go)). This is to make it easy for anyone contributing to Tilt itself to get HMR while working on the Tilt web UI.

If you're not actively hacking on the web UI but are using a source-built version of Tilt, you'll want to pass `--web-mode=prod`. That'll eliminate the need for yarn/node and substantially speed up startup since it won't need to bundle/minify the web UI assets.

(FWIW I personally avoid `go install`ing Tilt for that reason and let my `PATH` version be the official binary, and rely on `go run ./cmd/tilt` while developing.)

TL;DR For a non-Tilt-developer who installs a binary release of Tilt, the yarn/node requirement shouldn't exist -- it's not actually coming from the tutorial but from Tilt itself 🙃

**Comment by kensipe** [comment]
thank you for the awareness!  makes sense and closing!
